mordecai3/elastic_utilities.py

```diff
@@ -214,7 +214,7 @@ def _clean_search_name(search_name):
     return search_name
 
 def add_es_data(ex, conn, max_results=50, fuzzy=0, limit_types=False,
-                remove_correct=False, known_country=None):
+                remove_correct=False, include_countries:list[str] | None=None, exclude_countries:list[str] | None=None):
```

Comment:
Add spaces surrounding `=` of default values. Add type hints for other parameters.

I also generally suggest to move each parameter to its own line when you have a lot of them.

Reply:
Sure, done